src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -388,6 +388,13 @@ def _get_default_dns_prefix(name, resource_group_name, subscription_id):
     return '{}-{}-{}'.format(name_part, resource_group_part, subscription_id[0:6])
 
 
+def list_acs_locations():
+    return {
+        "Production Regions": regionsInProd,
+        "Preview Regions": regionsInPreview
```

Comment:
This should be camelcased.
Otherwise there would be spaces appear in the JSON output which would look strange I believe.